src/main/java/projectbookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,22 @@
 package projectbookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
```

Comment:
there is actually a specialized `@ISBN` for that purpose which we can use, and also I think that field is optional in our entity class, please double-check